src/site/content/en/blog/aspect-ratio/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: 'Maintaining Aspect Ratio in CSS'
+subhead: The new aspect-ratio property can help with responsive dimension-based imaged.
+authors:
+  - una
+date: 2021-01-14
+hero: hero.jpg
+alt: Image of colorful telephones resized with the same aspect ratio.
+description: There are several ways to maintain aspect ratio and image dimensions on the web. This post goes over two ways—one popular hack, and one new CSS property.
+tags:
+  - blog
+  - css
+---
+
+When resizing some media, in many cases, a design calls to maintain this ratio
+and ensure the image resizes in an even distribution. This width-to-height ratio
+is called the *aspect-ratio*.
+
+Aspect ratio is most commonly expressed as two integers and a colon in the
+dimensions of: width:height, or x:y. The most common aspect ratios for
+photography are 4:3 and 3:2, while video, and more recent consumer cameras, tend
+to have a 16:9 aspect ratio.
+
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="aspect-ratio.jpg" alt="">
+  <figcaption class="w-figcaption">
+    Two images with the same aspect ratio. One is 634 x 951px while the other is 200 x 300px. Both have a 2:3 aspect ratio.
+  </figcaption>
+</figure>
+
+## Why maintain aspect ratio
+
+With the advent of responsive design, maintaining aspect ratio has been
+increasingly important for web developers, especially as image dimensions
+differ. At times, it is difficult to maintain consistent layout sizing and place
+media like [video](http://fitvidsjs.com/) and background images within a parent
+block.
+
+## Object-fit
+
+One way to fit a media object into a parent is using the `object-fit` property,
+which enables users describe how content within a block should fill that block:
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="coffees.jpg" alt="">
+  <figcaption class="w-figcaption">
+    Showcasing various <code>object-fit</code> values. See <a href="https://codepen.io/una/pen/mdrLGjR">demo on Codepen</a>.
+  </figcaption>
+</figure>
+
+The `initial` and fill value will re-adjust the image to fill the space. In our
+example, this causes the image to be squished and blurry, as it re-adjusts
+pixels. This is not ideal. `object-fit: cover` uses the image's smallest
+dimension to fill the space and crops the image to fit into it based on this
+dimension. It “zooms in at its lowest boundary. `object-fit: contain` will
+ensure that the entire image is always visible, and so the opposite of `cover`:
+where it takes the largest boundary (in our example above this is width), and
+resize the image to maintain its intrinsic aspect-ratio while fitting into the
+space. The `object-fit: none` case shows the image cropped in it's center
+(default object position) at its natural size.
+
+`object-fit: cover` tends to work in most situations to ensure a nice uniform
+interface when dealing with images of varying dimensions, however, you lose data
```

Comment:
```suggestion
interface when dealing with images of varying dimensions, however, you lose information
```
I think we loosely mean information rather than data here

https://byjus.com/biology/difference-between-data-and-information/